projects/spectator/jest/src/lib/mock.ts

```diff
@@ -2,6 +2,7 @@
 
 import { FactoryProvider, AbstractType, Type } from '@angular/core';
 import { installProtoMethods, CompatibleSpy, SpyObject as BaseSpyObject } from '@ngneat/spectator';
+import { jest } from '@jest/globals'
```

Comment:
Is this a breaking change? It's from a specific version only or it'll throw in older jest versions?

Reply:
According to the [CHANGELOG.md](https://github.com/facebook/jest/blob/main/CHANGELOG.md#2550) from jest, it looks like that file appeared in version 25.5.0

Reply:
so it should be a peer dependency with this version and above. Please remove the release changes and the git tag. I will do it.

Reply:
Thanks!  I think I did what you were asking but not 100% sure.  I'm kinda new to giving PR's like this :)